src/infrastructure/channels/sequenced_channel.rs

```diff
@@ -0,0 +1,27 @@
+use super::Channel;
+
+use packet::PacketData;
+use infrastructure::DeliveryMethod;
+use error::NetworkResult;
+
+use std::io::Cursor;
+
+pub struct SequencedChannel {
+    reliable: bool,
+}
+
+impl SequencedChannel {
+    pub fn new(reliable: bool) -> SequencedChannel {
+        SequencedChannel { reliable }
+    }
+}
+
+impl Channel for SequencedChannel {
+    fn pre_process(&mut self, payload: &[u8], delivery_method: DeliveryMethod) -> NetworkResult<PacketData> {
+        unimplemented!()
```

Comment:
😄 lets open an issue for these or implement them.

Reply:
I'll maybe do that directly when this gets merged.